src/modules/DeleteTodo.js

```diff
@@ -0,0 +1,7 @@
+// export function handleDeleteBtnClick(event) {
+//   const { index } = event.target.dataset;
+//   deleteTask(index);
+//   renderTasks();
+// }
+
+// export default handleDeleteBtnClick;
```

Comment:
- [ ] 🙏Kindly remove the commented code in your project. This will make your code cleaner🗑.